blocks/aem-asset-selector/aem-asset-selector-util.js

```diff
@@ -63,6 +63,13 @@ const REL_RENDITIONS = 'http://ns.adobe.com/adobecloud/rel/rendition';
 // TODO: change this to Asset Link IMS client ID
 const IMS_CLIENT_ID = 'p66302-franklin';
 const ASSET_SELECTOR_ID = 'asset-selector';
+const CLIPBOARD_SUPPORTED_BINARY_MIMETYPES = [
+  'image/png',
+];
+const SUPPORTED_RENDITIONS_FORMATS = [
```

Comment:
Renditions mimetypes that we can use in Franklin documents (needed for filtering).